packages/storybook/src/example-pages/Patronen/card-panel.ts

```diff
@@ -0,0 +1,124 @@
+import { html } from 'lit-html';
+
+import { examplePageFactory } from '../../example-page-factory';
+
+examplePageFactory(
+  'patronen',
+  'Card Panel',
+  ({ searchBarTemplate, anchorTemplate, buttonTemplate, imageTemplate, highlightBoxTemplate, toggletipTemplate, labelTemplate, viewerGridTemplate }) => html`
+    <div class="container">
+      ${viewerGridTemplate({ initialMainSize: 'large', map: html``, main: html`
+        <h1>Overzicht op uw locatie</h1>
+          ${highlightBoxTemplate({ white: true, dropShadow: true, content: html`
+            <div class="dso-rich-content">
+              <h2>Activiteiten</h2>
+              <p>Als u wilt bouwen of aanpassen kunt u regels zoeken voor deze activiteit.</p>
+            </div>
+            ${searchBarTemplate({ buttonLabel: 'Zoekopdracht legen', id: 'PatronenCardPanelSearchBarID', label: 'Label van de search-bar', placeholder: 'Placeholder van de search-bar', hiddenLabel: true, icon: true, hideSearchButton: true })}
+            <div class="dso-rich-content">
+              <p>
+                Veel gebruikt:
+                ${anchorTemplate({ label: 'Boom kappen', url: '#' })},
+                ${anchorTemplate({ label: 'Dakkapel plaatsen', url: '#' })},
+                ${anchorTemplate({ label: 'Schuur bouwen', url: '#' })}
+              </p>
+            </div>
+
+            <div class="dso-context-wrapper">
```

Comment:
@MichielDHVD Hier doen we iets niet goed. Nu wordt het Context component (`dso-context-wrapper`) misbruikt om een knop rechts uit te lijnen, terwijl het component ook vraagt om een `label` en `children`